tests/arch/riscv/test_build_cpp/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 Google Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(test_build_cpp)
```

Comment:
If we are going to add such test, it should probably be arch-agnostic

Reply:
See latest changes.